ui/apps/dashboard/src/components/Secret/Secret.tsx

```diff
@@ -23,27 +25,44 @@ export function Secret({ className, kind, secret }: Props) {
   return (
     <div
       className={cn(
-        'flex rounded-md border border-slate-300 bg-slate-50 text-slate-500',
+        'flex truncate rounded-md border border-slate-300 bg-slate-50 text-slate-500',
```

Comment:
```suggestion
        'flex rounded-md border border-slate-300 bg-slate-50 text-slate-500',
```

Reply:
Then what you need there is an `overflow-hidden`! The truncate tailwind class includes it, but it's not the most adequate, since it's a text overflow utility.